api/capi/include/nnstreamer-single.h

```diff
@@ -185,6 +185,33 @@ int ml_single_set_input_info (ml_single_h single, const ml_tensors_info_h info);
  */
 int ml_single_set_timeout (ml_single_h single, unsigned int timeout);
 
+/**
+ * @brief Sets the property value for the given model.
+ * @details Note that a model/framework may not support to change the property after opening the model.
```

Comment:
How about update `opening the model` to `after the first call to invoke`? The user will not know when the model is opened. It might seem that `ml_single_open` might open the model.
Before the first invoke call, the properties can be updated freely as the framework is not yet open.
However, after the first invoke (ml_single_invoke), the support of updating the properties will be limited depending on the support provided by the framework.

Reply:
ml_single_open() calls the callback start(), then open the model. So I think 'open model' is ok.

Reply:
aah, yes, my bad.